libr/util/utype.c

```diff
@@ -99,7 +99,7 @@ R_API char *r_type_enum_getbitfield(Sdb *TDB, const char *name, ut64 val) {
 	bool isFirst = true;
 	ret = r_str_appendf (ret, "0x%08"PFMT64x" : ", val);
 	for (i = 0; i < 32; i++) {
-		if (!(val & (1 << i))) {
+		if (!(val & ((ut32)1 << i))) {
```

Comment:
Wouldn't it be simpler to use the appropriate unsigned integer literal `1u` instead of cast? Although in the case shifting gets expanded to 64 it's more obvious that `(ut32)` cast needs to be replaced `(ut64)` than changing `1u` to `1ull`. The decision is up to you.

Reply:
yeah agree, i fixed the thing. chek it now